### ISSUE TITLE
[BUG] Installation, quickstart and documentation out of date.

### ISSUE THREAD

**Issue by BlakeList** [deepchecks/deepchecks#1042]
**Describe the bug**
When installing deepchecks on Python 3.7 with

`pip install --upgrade deepchecks`

Then running imports like:

`from deepchecks.checks import TrainTestFeatureDrift`
or
`from deepchecks.suites import full_suite`
yields:
`AttributeError: module 'pandas' has no attribute 'Panel'`

See below for full error message.

Further, the documentation [quickstart for tabular data](https://docs.deepchecks.com/en/latest/tutorials/tabular/quickstart_in_5_minutes.html) specifies running:

`from deepchecks.tabular import Dataset`
and 
`from deepchecks.tabular.suites import full_suite`
which result in:
`ModuleNotFoundError: No module named 'deepchecks.tabular'`

**Environment (please complete the following information):**
 - OS: CentOS Linux release 7.9.2009 (Core)
 - Python Version: 3.7.10
 - Deepchecks Version: 0.5.0

**Additional context**
Full error message:
`
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-a0cb083fbfb6> in <module>
     10 
     11 # # import deepchecks as dc
---> 12 from deepchecks.checks import TrainTestFeatureDrift

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/deepchecks/checks/__init__.py in <module>
     25 )
     26 
---> 27 from .methodology import (
     28     BoostingOverfit,
     29     UnusedFeatures,

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/deepchecks/checks/methodology/__init__.py in <module>
     18 from .date_train_test_leakage_duplicates import DateTrainTestLeakageDuplicates
     19 from .date_train_test_leakage_overlap import DateTrainTestLeakageOverlap
---> 20 from .identifier_leakage import IdentifierLeakage
     21 from .model_inference_time import ModelInferenceTime
     22 from .datasets_size_comparison import DatasetsSizeComparison

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/deepchecks/checks/methodology/identifier_leakage.py in <module>
     12 from typing import Dict
     13 
---> 14 import plotly.express as px
     15 
     16 import deepchecks.ppscore as pps

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/plotly/express/__init__.py in <module>
     13     )
     14 
---> 15 from ._imshow import imshow
     16 from ._chart_types import (  # noqa: F401
     17     scatter,

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/plotly/express/_imshow.py in <module>
      9 
     10 try:
---> 11     import xarray
     12 
     13     xarray_imported = True

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/xarray/__init__.py in <module>
     10 from .core.alignment import align, broadcast, broadcast_arrays
     11 from .core.common import full_like, zeros_like, ones_like
---> 12 from .core.combine import concat, auto_combine
     13 from .core.computation import apply_ufunc, dot, where
     14 from .core.extensions import (register_dataarray_accessor,

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/xarray/core/combine.py in <module>
      9 from . import utils
     10 from .alignment import align
---> 11 from .merge import merge
     12 from .pycompat import OrderedDict, basestring, iteritems
     13 from .variable import IndexVariable, Variable, as_variable

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/xarray/core/merge.py in <module>
      8 from .variable import as_variable, assert_unique_multiindex_level_names
      9 
---> 10 PANDAS_TYPES = (pd.Series, pd.DataFrame, pd.Panel)
     11 
     12 _VALID_COMPAT = Frozen({'identical': 0,

/workspace/appscratch/miniconda/cflbwl_black/lib/python3.7/site-packages/pandas/__init__.py in __getattr__(name)
    242         return _SparseArray
    243 
--> 244     raise AttributeError(f"module 'pandas' has no attribute '{name}'")
    245 
    246 

AttributeError: module 'pandas' has no attribute 'Panel'
`

**Comment by ItayGabbay** [comment]
Hi @BlakeList 

Thanks for your issue!

- regarding the error while import 'from deepchecks.suites import full_suite' - can you please attach your plotly and pandas versions?

- regarding the not updated docs, I see that you attached a link from the 'latest' version - which does not reflect the latest stable release on pypi. Please try to view the docs version at the stable version https://docs.deepchecks.com/en/stable/ 

Since we will soon release the computer vision module, the structure in the package was changed a lot and the latest release of the docs reflects the main branch.

**Comment by BlakeList** [comment]
Hi @ItayGabbay

The plotly and pandas versions are installed from what is specified in the Deepchecks requirements (plotly>=5.5.0, pandas>=1.1.5), which are currently:
`pandas==1.3.5` and
`plotly==5.6.0`

I see that the stable docs have the code:
`from deepchecks.suites import full_suite`, highlighting the first issue. Also, I am unable to find the current [list of existing suites](https://github.com/deepchecks/deepchecks/tree/main/deepchecks/suites)

**Comment by ItayGabbay** [comment]
Ok, got you. Thanks.
While we work on the hotfix, can you please try to downgrade the plotly version to 5.5.0?

**Comment by BlakeList** [comment]
Downgrading the plotly version to 5.5.0 and running 
`from deepchecks.suites import full_suite`
still results in
`AttributeError: module 'pandas' has no attribute 'Panel'`

**Comment by ItayGabbay** [comment]
Thank you @BlakeList .

For the list of existing suites - you can view them [here](https://docs.deepchecks.com/en/stable/api/generated/deepchecks.suites.default_suites.html#module-deepchecks.suites.default_suites), or [here](https://github.com/deepchecks/deepchecks/tree/0.5.x/deepchecks/suites).
Thanks to this issue, we will fix the versioning mismatch in the documentation in the upcoming release

Regarding the import bug - I cannot reproduce it in my env. It will be really helpful if you can share the output of your `pip freeze` so we'll be able to investigate further. 

Did you install deepchecks on a clean virtualenv?

**Comment by BlakeList** [comment]
Hi @ItayGabbay 

Thanks for the links.

I have clean installed in a conda env and the import works, with pandas==1.3.5, plotly==5.6.0.

I was previously installing into a general work dev environment which has many other packages, but still the same versions of pandas and plotly.

Any idea why this wouldn't work when the error is referencing pandas and the versions are the same?

I have added both pip freezes below.

Clean env:

argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
attrs==21.4.0
backcall==0.2.0
beautifulsoup4==4.10.0
bleach==4.1.0
category-encoders==2.4.0
certifi==2021.10.8
cffi==1.15.0
click==7.1.2
cloudpickle==1.6.0
croniter==0.3.37
cycler==0.11.0
debugpy==1.4.1
decorator==5.0.9
deepchecks==0.5.0
defusedxml==0.7.1
docker==5.0.0
entrypoints==0.4
fonttools==4.31.1
HeapDict==1.0.1
importlib-metadata==3.10.1
importlib-resources==5.4.0
ipykernel==6.9.2
ipython==7.32.0
ipython-genutils==0.2.0
ipywidgets==7.7.0
jedi==0.18.0
Jinja2==3.0.3
joblib==1.1.0
jsonpickle==2.1.0
jsonschema==4.4.0
jupyter-client==6.1.12
jupyter-core==4.7.1
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.1.0
kiwisolver==1.4.0
locket==0.2.1
MarkupSafe==2.1.1
marshmallow==3.13.0
marshmallow-oneofschema==3.0.1
matplotlib==3.5.1
matplotlib-inline==0.1.2
mistune==0.8.4
msgpack==1.0.2
mypy-extensions==0.4.3
natsort==7.1.1
nbclient==0.5.13
nbconvert==6.4.4
nbformat==5.2.0
nest-asyncio==1.5.4
notebook==6.4.10
numpy==1.21.5
packaging==21.3
pandas==1.3.5
pandocfilters==1.5.0
parso==0.8.2
partd==1.2.0
patsy==0.5.2
pendulum==2.1.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
plotly==5.6.0
prometheus-client==0.13.1
prompt-toolkit==3.0.28
psutil==5.8.0
ptyprocess==0.7.0
pycparser==2.21
Pygments==2.9.0
pyparsing==2.4.7
pyrsistent==0.18.1
python-box==5.3.0
python-dateutil==2.8.2
python-slugify==5.0.2
pytz==2021.1
pytzdata==2020.1
pyzmq==22.1.0
scikit-learn==1.0.2
scipy==1.7.3
Send2Trash==1.8.0
six==1.16.0
sortedcontainers==2.4.0
soupsieve==2.3.1
statsmodels==0.13.2
tabulate==0.8.9
tblib==1.7.0
tenacity==8.0.1
terminado==0.13.3
testpath==0.6.0
text-unidecode==1.3
threadpoolctl==3.1.0
toml==0.10.2
toolz==0.11.1
tornado==6.1
tqdm==4.63.0
traitlets==5.1.1
typing_extensions==4.1.1
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.1.0
widgetsnbextension==3.6.0
zict==2.0.0
zipp==3.5.0


Dev env:

absl-py==0.12.0
-e git+https://github.com/arangoml/dgl-adapter.git@c172daf1a7be4dc738866e09012c2736f9eaa85c#egg=adbdgl_adapter&subdirectory=adbdgl_adapter
adbnx-adapter==3.0.1
aiohttp==3.7.4.post0
aiohttp-retry==2.4.6
aioredis==1.3.1
alabaster==0.7.12
alembic==1.4.1
altair==4.1.0
ansiwrap==0.8.4
appdirs==1.4.4
arangodb==0.0.12
arff==0.9
arrow==0.15.6
arviz==0.7.0
astor==0.8.0
astroid==2.8.0
astroML==0.4.1
astropy==4.0.1.post1
astunparse==1.6.3
async-generator==1.10
async-timeout==3.0.1
asyncssh==2.8.1
atpublic==2.3
attrs @ file:///tmp/build/80754af9/attrs_1604765588209/work
autoflake==1.4
autopy==4.0.0
awscli==1.20.54
azure-core==1.7.0
azure-storage-blob==12.3.2
Babel==2.7.0
backcall==0.2.0
backports.zoneinfo==0.2.1
base58==2.1.0
bayespy==0.5.18
beautifulsoup4==4.8.1
beautifultable==0.8.0
bel-resources==0.0.3
bertopic==0.9.4
bioc==1.3.6
bioregistry==0.0.13
bioversions==0.0.10
bitarray==2.1.0
black==21.9b0
bleach==3.1.0
blessings==1.7
blinker==1.4
blis==0.7.4
bokeh==2.0.2
boto==2.49.0
boto3==1.9.253
botocore==1.21.54
bpemb==0.3.2
Brotli==1.0.9
brotlipy==0.7.0
cached-property==1.5.2
cachetools==4.0.0
cachier==1.5.0
catalogue==2.0.6
category-encoders==2.4.0
certifi==2020.12.5
cffi @ file:///tmp/build/80754af9/cffi_1613246939562/work
cftime==1.1.1.2
chainer==7.4.0
chardet==3.0.4
charset-normalizer==2.0.6
chart-studio==1.1.0
chembl-webresource-client==0.10.3
cliar==1.3.2
click==7.1.2
click-default-group==1.2.2
click-plugins==1.1.1
cloudpickle==1.6.0
colorama==0.4.3
colorcet==2.0.2
colorful==0.5.4
colorlover==0.3.0
colorspace @ git+https://github.com/retostauffer/python-colorspace@f823acded12f74c16095daa7c0a33907ee63f62c
comcrawl==1.0.2
commonmark==0.9.1
configobj==5.0.6
confuse==1.3.0
conllu==4.4
costcla==0.6.dev0
costsensitive==0.1.2.10
croniter==0.3.37
cryptography @ file:///tmp/build/80754af9/cryptography_1616769182610/work
cssselect==1.1.0
cufflinks==0.17.3
cycler==0.10.0
cymem==2.0.5
Cython==0.29.21
cytoolz==0.9.0.1
dash==1.16.0
dash-core-components==1.12.0
dash-html-components==1.1.1
dash-renderer==1.8.0
dash-table==4.10.1
dask==2.30.0
databricks-cli==0.11.0
dataclasses-json==0.5.2
datashader==0.10.0
datashape==0.5.2
datatable==1.0.0
deap==1.3.0
debugpy==1.4.1
decorator==5.0.9
deepchecks @ git+https://github.com/deepchecks/deepchecks@66a651404c747a9767cf15ea5fb5b2eb3ae1d8d6
deepchem @ file:///home/conda/feedstock_root/build_artifacts/deepchem_1616192315460/work
deepsnap==0.2.1
defusedxml==0.6.0
Deprecated==1.2.9
dgl==0.6.1
dictdiffer==0.9.0
dill==0.2.9
diskcache==5.2.1
distlib==0.3.1
distributed==2.30.1
distro==1.6.0
docker==5.0.0
docstringtest==0.3.0
docutils==0.15.2
dpath==2.0.5
drugbank-downloader==0.0.3
dulwich==0.20.25
dvc==2.9.3
easydict==1.9
elasticsearch==7.1.0
eli5==0.11.0
en-core-sci-lg @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_lg-0.4.0.tar.gz
en-core-sci-scibert @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_scibert-0.4.0.tar.gz
en-core-sci-sm @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.3.0/en_core_sci_sm-0.3.0.tar.gz
en-core-web-lg @ https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.2.0/en_core_web_lg-3.2.0-py3-none-any.whl
en-core-web-md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.2.0/en_core_web_md-3.2.0-py3-none-any.whl
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.2.0/en_core_web_sm-3.2.0-py3-none-any.whl
en-core-web-trf @ https://github.com/explosion/spacy-models/releases/download/en_core_web_trf-3.2.0/en_core_web_trf-3.2.0-py3-none-any.whl
en-coref-md @ https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_md-3.0.0/en_coref_md-3.0.0.tar.gz
en-ner-bionlp13cg-md @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bionlp13cg_md-0.4.0.tar.gz
entrezpy==2.1.3
entrypoints==0.3
et-xmlfile==1.1.0
fake-useragent==0.1.11
Faker==6.5.0
fasteners==0.15
fastobo==0.11.1
fastparquet==0.3.3
fastprogress==1.0.0
fasttext==0.9.1
filelock==3.0.12
flair==0.7
flake8==3.9.2
flashtext==2.7
Flask==1.1.1
Flask-Compress==1.5.0
flatbuffers==1.12
flatten-dict==0.4.2
flufl.lock==6.0
foliant==1.0.11
foliantcontrib.imagemagick==1.0.2
fonttools==4.28.5
fsspec==2021.11.1
ftfy==5.8
funcy==1.14
future==0.18.2
gast==0.4.0
gdown==3.12.2
gensim==3.8.3
gitdb==4.0.5
GitPython==3.1.7
google==3.0.0
google-api-core==1.26.2
google-auth==1.28.0
google-auth-oauthlib==0.4.1
google-compute-engine==2.8.13
google-pasta==0.2.0
googleapis-common-protos==1.51.0
googledrivedownloader==0.4
gorilla==0.3.0
gpustat==0.6.0
grandalf==0.6
graphviz==0.14.1
great-expectations==0.13.31
greenlet @ file:///tmp/build/80754af9/greenlet_1611958357961/work
grpcio==1.34.1
-e git+https://github.com/vi3k6i5/GuidedLDA@6ddfbe47b5f3f8138852c1987cc55bc0703c8f8b#egg=guidedlda
gunicorn==20.0.4
h5py==3.1.0
hdbscan==0.8.27
HeapDict==1.0.1
hiredis==1.1.0
holoviews==1.13.2
html5lib==1.0.1
htmlmin==0.1.12
huggingface-hub==0.0.12
humanize==3.2.0
hvplot==0.7.3
hyperopt==0.2.3
hypothesis==6.10.1
idna @ file:///home/linux1/recipes/ci/idna_1610986105248/work
imagecodecs==2020.2.18
ImageHash==4.1.0
imageio==2.8.0
imagesize==1.1.0
imgkit==1.2.2
importlib-metadata==3.10.1
importlib-resources==5.4.0
inexactsearch==1.0.2
install==1.3.3
interchange==2021.0.4
intervaltree==3.1.0
ipykernel==5.5.6
ipython==7.1.1
ipython-genutils==0.2.0
ipywidgets==7.5.1
isodate==0.6.0
isort==5.9.3
itsdangerous==1.1.0
Janome==0.3.10
jedi==0.18.0
jeepney==0.7.0
jellyfish==0.7.2
Jinja2==2.11.2
jmespath==0.9.4
joblib @ file:///tmp/build/80754af9/joblib_1613502643832/work
jsonlines==1.2.0
jsonpatch==1.32
jsonpickle==2.0.0
jsonpointer==2.1
jsonschema @ file:///tmp/build/80754af9/jsonschema_1602607155483/work
jupyter==1.0.0
jupyter-client==6.1.12
jupyter-console==6.0.0
jupyter-core==4.7.1
keras==2.7.0
Keras-Applications==1.0.8
keras-contrib @ git+https://www.github.com/keras-team/keras-contrib.git@3fc5ef709e061416f4bc8a92ca3750c824b5d2b0
keras-nightly==2.5.0.dev2021032900
Keras-Preprocessing==1.1.2
keyring==23.0.1
kindred==2.7.8
kiwicloudpy==1.0rc2
kiwisolver @ file:///tmp/build/80754af9/kiwisolver_1612282414123/work
konoha==4.6.2
langcodes==3.3.0
langdetect==1.0.8
latexcodec==1.0.7
lazy-object-proxy==1.6.0
lda==1.1.0
lda2vec==0.16.10
libclang==12.0.0
libcst==0.3.23
lightning-python==1.2.1
lime==0.2.0.1
linecache2==1.0.0
llvmlite==0.35.0
locket==0.2.1
lxml==4.4.1
mailchecker==4.0.12
Mako==1.1.3
Markdown==3.1.1
MarkupSafe==2.0.1
marshmallow==3.13.0
marshmallow-enum==1.5.1
marshmallow-oneofschema==3.0.1
matplotlib==3.5.1
matplotlib-inline==0.1.2
mccabe==0.6.1
missingno==0.4.2
mistune==0.8.4
mittens==0.2
mkl-fft==1.0.14
mkl-random==1.0.4
mkl-service==2.3.0
ml-datasets==0.2.0
mlflow==1.24.0
mlxtend==0.17.0
mock==2.0.0
modulecmd==1.1.3
monotonic==1.5
more-click==0.0.3
more-itertools==7.2.0
mpld3==0.3
msgpack==1.0.2
msgpack-numpy==0.4.4
msrest==0.6.18
MulticoreTSNE==0.1
multidict==5.1.0
multimethod==1.6
multipledispatch==0.6.0
multisplitby==0.0.1
munkres==1.1.4
murmurhash==1.0.5
mypy-extensions==0.4.3
names==0.3.0
nanotime==0.5.2
natsort==7.1.1
nbclient==0.5.3
nbconvert==5.6.0
nbformat @ file:///tmp/build/80754af9/nbformat_1617383369282/work
nbsphinx==0.4.3
nest-asyncio==1.5.1
netCDF4==1.5.3
netgraph==4.2.3
netpbm==0.0.1
networkx==2.6.3
neuralcoref==4.0
nltk==3.4.5
nmslib==2.0.6
nose==1.3.7
notebook==6.0.1
ntlm-auth==1.4.0
numba==0.52.0
numexpr==2.7.1
numpy==1.21.5
numpydoc==0.9.1
nvidia-ml-py3==7.352.0
oauthlib==3.1.0
obonet==0.2.6
olefile==0.46
onnxruntime==1.7.0
opencensus==0.7.12
opencensus-context==0.1.2
openpyxl==3.0.7
opt-einsum==3.3.0
oset==0.1.3
overrides==3.0.0
Owlready2==0.36
packaging==21.3
pandas==1.3.5
pandas-profiling==3.1.0
pandocfilters==1.4.2
panel==0.9.5
pansi==2020.7.3
papermill==2.3.3
param==1.9.3
parso==0.8.2
partd==1.2.0
pathlib==1.0.1
pathspec==0.9.0
pathtools==0.1.2
pathy==0.4.0
patsy==0.5.2
pbr==3.1.1
pc-toolbox==0.1.20180712
pdfminer==20191125
pendulum==2.1.2
pexpect==4.8.0
phik==0.12.0
phonenumbers==8.12.34
phytebyte @ git+https://github.com/seanharr11/phytebyte.git@1f3f0ac2a09da9f9b45741bbc0ef3eefb40b3f03
pickle-mixin==1.0.2
pickle5==0.0.11
pickleshare==0.7.5
Pillow @ file:///tmp/build/80754af9/pillow_1617386154241/work
pipenv==2020.11.15
plac==0.9.6
platformdirs==2.4.0
plotly==5.6.0
pluggy==0.13.1
ply==3.11
portalocker==2.2.1
prefect==0.15.13
preshed==3.0.5
prince==0.7.1
proj==0.2.0
prometheus-client==0.7.1
prometheus-flask-exporter==0.15.4
promise==2.3
prompt-toolkit==2.0.10
pronto==2.4.4
protmapper==0.0.20
protobuf==3.15.6
psutil==5.8.0
psycopg2==2.7.7
psycopg2-binary==2.9.1
PTable==0.9.2
ptyprocess==0.7.0
PubChemPy==1.0.4
PuLP==1.6.10
py==1.8.0
py-spy==0.3.5
py2neo==2021.2.3
pyarrow==5.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pybind11==2.5.0
pybtex==0.22.2
pybtex-docutils==0.2.2
pycairo==1.19.1
PyChromeDevTools @ git+https://github.com/marty90/PyChromeDevTools@23fa4ff9f65d9c22596449481011324824fe424b
pycodestyle==2.7.0
pycparser @ file:///tmp/build/80754af9/pycparser_1594388511720/work
pycryptodome==3.9.9
pyct==0.4.6
pydantic==1.8.2
pydeck==0.6.1
pydot==1.4.2
pyea==0.2
pyemd==0.5.1
pyflakes==2.3.1
pygit2==1.7.0
Pygments==2.9.0
pygtrie==2.4.2
PyJWT==2.3.0
pykakapo @ file:///powerplant/workspace/cflbwl/github/NLP-Applications/pykakapo
pyLDAvis==2.1.2
pylint==2.11.1
pymc==2.3.7
pymc3==3.8
Pympler==1.0.1
pynndescent==0.5.2
pyobo==0.3.3
pyOpenSSL @ file:///tmp/build/80754af9/pyopenssl_1608057966937/work
pyparsing==2.4.7
PyPDF2==1.26.0
Pyphen==0.9.5
PyQt5==5.15.1
PyQt5-sip==12.8.1
pyquery==1.4.1
pyro-api==0.1.2
pyro-ppl==1.3.1
Pyro4==4.80
pyrsistent @ file:///tmp/build/80754af9/pyrsistent_1600141707582/work
pysbd==0.3.3
PySocks @ file:///tmp/build/80754af9/pysocks_1594394576006/work
PyStemmer==1.3.0
pystow==0.0.10
pytest==5.4.1
pytest-socket==0.3.5
pytest-timeout==1.4.2
python-arango==7.2.0
python-benedict==0.24.3
python-box==5.3.0
python-dateutil==2.8.2
python-editor==1.0.4
python-fsutil==0.5.0
python-Levenshtein==0.12.2
python-slugify==5.0.2
pytokenizations==0.7.2
pytorch-lamb==1.0.0
pytorch-pretrained-bert==0.6.2
pytz==2021.1
pytzdata==2020.1
pyupgrade==2.29.1
pyvis==0.1.9
pyviz-comms==0.7.4
PyWavelets==1.1.1
PyYAML==6.0
pyzmq==22.1.0
qtconsole==4.5.5
querystring-parser==1.2.4
rake-nltk==1.0.4
ratelimit==2.2.1
ray==0.8.7
rdflib==4.2.2
recommonmark==0.6.0
rectangle-packer==2.0.1
redis==3.4.1
regex==2021.4.4
reportlab==3.5.67
requests==2.27.1
requests-cache==0.6.2
requests-file==1.5.1
requests-ftp==0.3.1
requests-ntlm==1.1.0
requests-oauthlib==1.3.0
requests-toolbelt==0.9.1
retrying==1.3.3
rich==10.16.2
rise==5.7.1
rsa==4.0
ruamel.yaml==0.17.16
ruamel.yaml.clib==0.2.2
s3transfer==0.5.0
sacremoses==0.0.38
scattertext==0.1.2
scikit-image==0.17.2
scikit-learn==0.24.2
scikit-multilearn==0.2.0
scipy==1.7.3
scispacy==0.4.0
scmrepo==0.0.7
seaborn==0.11.2
SecretStorage==3.3.1
segtok==1.5.9
selenium==3.141.0
semanticscholar==0.1.6
Send2Trash==1.5.0
sentence-transformers==2.2.0
sentencepiece==0.1.91
seqeval==0.0.12
serpent==1.30.2
setuptools-scm==6.3.2
Shapely==1.7.0
shed==0.5.3
shorttext==1.1.5
shortuuid==1.0.1
shtab==1.4.2
silpa-common==0.3
sister==0.1.5
six @ file:///tmp/build/80754af9/six_1605205313296/work
sklearn==0.0
sklearn-contrib-lightning==0.6.0
smart-open==3.0.0
smmap==3.0.4
smop==0.41
snorkel==0.9.7
snowballstemmer==2.0.0
sortedcontainers==2.4.0
soundex==1.1.3
soupsieve==1.9.5
spacy==3.2.3
spacy-alignments==0.7.2
spacy-legacy==3.0.8
spacy-loggers==1.0.1
spacy-ray==0.1.0
spacy-streamlit==1.0.2
spacy-transformers==1.1.4
spark-nlp==2.6.1
spellchecker==0.4
Sphinx==1.5.5
sphinx-rtd-theme==0.4.3
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-bayesnet==0.1
sphinxcontrib-bibtex==1.0.0
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.5
sphinxcontrib-tikz==0.4.8
spynner @ git+https://github.com/kiorky/spynner.git@882cf3f40b900fbe52f895edf0ac18b4db068038
SQLAlchemy @ file:///tmp/build/80754af9/sqlalchemy_1618089545514/work
sqlitedict==1.6.0
sqlparse==0.3.1
srsly==2.4.2
stanford-openie==1.0.1
stanfordcorenlp==3.9.1.1
stanfordnlp==0.2.0
statsmodels==0.13.2
streamlit==1.3.1
streamlit-pandas-profiling==0.1.2
streamz==0.6.3
stringcase==1.2.0
supervisor==4.2.4
tables==3.6.1
tabula-py==2.3.0
tabulate==0.8.9
tangled-up-in-unicode==0.1.0
taumahi==1.2
tblib==1.7.0
tenacity==8.0.1
tensorboard==2.7.0
tensorboard-data-server==0.6.0
tensorboard-plugin-wit==1.7.0
tensorboardX==2.1
tensorflow==2.7.0
tensorflow-datasets==2.0.0
tensorflow-estimator==2.7.0
tensorflow-gpu==2.7.0
tensorflow-io-gcs-filesystem==0.21.0
tensorflow-metadata==0.21.1
termcolor==1.1.0
terminado==0.8.2
terminusdb-client==10.0.23
testfixtures==6.14.1
testpath==0.4.2
text-unidecode==1.3
textacy==0.10.0
textblob==0.15.3
textwrap3==0.9.2
Theano==1.0.4
thefuzz==0.19.0
thinc==8.0.13
threadpoolctl @ file:///tmp/tmp9twdgx9k/threadpoolctl-2.1.0-py3-none-any.whl
thrift==0.13.0
tifffile==2020.5.11
tika==1.24
tokenize-rt==4.2.1
tokenizers==0.10.3
toml==0.10.2
tomli==1.2.1
toolz==0.11.1
torch==1.10.1
torch-cluster==1.5.9
torch-geometric==2.0.3
torch-scatter==2.0.9
torch-sparse==0.6.12
torch-spline-conv==1.2.1
torchcontrib==0.0.2
torchmetrics==0.3.0
torchvision==0.11.2
tornado==6.1
tqdm==4.63.0
traceback2==1.4.0
traitlets @ file:///home/ktietz/src/ci/traitlets_1611929699868/work
transformers==4.9.1
typed-ast==1.4.3
typeguard==2.13.2
typer==0.3.2
typing-extensions==3.10.0.2
typing-inspect==0.6.0
tzlocal==3.0b1
ujson==4.0.1
umap-learn==0.5.1
Unidecode==1.1.2
unittest2==1.1.0
url-normalize==1.4.3
urllib3==1.26.7
utils==1.0.1
validators==0.18.2
virtualenv==20.4.6
virtualenv-clone==0.5.4
visions==0.7.4
voluptuous==0.12.2
waitress==1.4.4
wasabi==0.8.2
watchdog==2.0.2
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.1.0
Werkzeug==0.16.0
wget==3.2
widgetsnbextension==3.5.1
wrapt==1.12.1
xarray==0.11.3
xlrd==1.2.0
xmltodict==0.12.0
yacs==0.1.8
yake @ git+https://github.com/LIAAD/yake@2c9fedbe0e3195a273a3c3daef3d41d4d7c5b3b3
yarl==1.6.3
yellowbrick==1.2
yelp-bytes==0.3.0
yelp-encodings==0.1.3
yelp-uri==2.0.0
zc.lockfile==2.0
zict==2.0.0
zipp==3.5.0

**Comment by ItayGabbay** [comment]
Hi @BlakeList 
Thank you!

From my understanding it's related to the xarray package, probably there's a mismatch between its version and the version plotly requires.

**Comment by BlakeList** [comment]
Hi @ItayGabbay 

Firstly, thank you for the quick replies.

I have managed to get deepchecks importing properly on the general dev env. The solution was to:
`pip uninstall xarray pandas plotly deepchecks` then 
`pip install --upgrade deepchecks` as this allowed the correct versions to be installed.

Feel free to close, and anyone else with problems can reference this issue.

Looking forward to playing with deepchecks!

Kind regards,
Blake

**Comment by ItayGabbay** [comment]
I am happy it's working now :) closing this issue. We'll be glad to hear some feedback when you try it!
